docs/docs/mcp/mcp-registry.md

```
@@ -0,0 +1,467 @@
+
+```yaml
+mcpServers:
+  filesystem:
+    type: stdio
+    command: npx
+    args: ["-y", "@modelcontextprotocol/server-filesystem", "."]
+    timeout: 30000
+```
+
+**Tags:** file, directory, filesystem, io
+**Homepage:** https://github.com/modelcontextprotocol/servers
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)